site/docs/plugins/i18n.md

```diff
@@ -617,6 +617,20 @@ bot.command("start", async (ctx) => {
 });
 ```
 
+::: warning Potential Formatting Issues
+By default, Fluent uses Unicode isolation marks for interpolations. 
```

Comment:
This is the first time I've heard this term. This link may help readers who are also new to the topic.
```suggestion
By default, Fluent uses [Unicode isolation](https://github.com/projectfluent/fluent.js/wiki/Unicode-Isolation) marks for interpolations. 
```